parslepy/base.py

```diff
@@ -6,7 +6,7 @@
 import lxml.html
 import re
 import json
-
+import yaml
```

Comment:
As `yaml` is not a mandatory dependency, what do you think of moving the import statement inside `from_yamlstring`?